docs/Development/ReleaseNotes.md

```diff
@@ -91,6 +91,10 @@ To check which release of VIC you are running:
 
 	Codified behavior in the initialization of the ``image`` and `cesm` drivers that requires the parameter variables `AreaFract`, `Pfactor`, `zone_fract`, and `Cv` must sum exactly to 1.0. If using the `SNOW_BAND` option, the area weighted `elevation` must match the mean grid cell elevation (`elev`). VIC will print *warnings* if any of these criteria are violated.  
 
+7. Added thread parallelization using OPENMP ([GH#712](https://github.com/UW-Hydro/VIC/pull/712))
```

Comment:
Yes, here's an informal test run on my iMac for a 10day test. 

| Tests     | No MPI, No OpenMP | No MPI, 2 OpenMP Threads | No MPI, 4 OpenMP Threads | 2 MPI tasks, No OpenMP | 4 MPI task, No OpenMP | 2 MPI task, 2 OpenMP Threads |
|-----------|-------------------|--------------------------|--------------------------|------------------------|-----------------------|------------------------------|
| Wall Time | 8.03972           | 5.3995                   | 3.91444                  | 4.72344                | 3.71287               | 3.10787                      |
| CPU Time  | 6.80228           | 6.67521                  | 6.52599                  | 7.87946                | 9.50568               | 7.23022                      |